src/order/pfilter.lean

```diff
@@ -65,6 +69,8 @@ instance : has_coe (pfilter P) (set P) := ⟨λ F, F.dual.carrier⟩
 /-- For the notation `x ∈ F`. -/
 instance : has_mem P (pfilter P) := ⟨λ x F, x ∈ (F : set P)⟩
 
+@[simp] lemma mem_def : x ∈ F ↔ x ∈ (F : set P) := iff_of_eq rfl
```

Comment:
I think the direction of this simp lemma should be swapped (and the lemma should be called `mem_coe`).
That is what we have for `submodule` and many other bundled sets.